packages/popover/src/positionUtils.ts

```diff
@@ -103,8 +114,29 @@ const defaultElementPosition = {
   width: 0,
 };
 
+export function getElementDocumentPosition(
+  element: HTMLElement | null,
+): ElementPosition {
+  if (!element) {
+    return defaultElementPosition;
+  }
+
+  const { top, bottom, left, right } = element.getBoundingClientRect();
+  const { offsetHeight: height, offsetWidth: width } = element;
```

Comment:
Doesn't `getBoundingClientRect` also return `height` and `width`?

Reply:
Oh but this was probably copy/paste and not new. So whatever :)

Reply:
Yeah, this component could use a bit of cleaning up, but I was trying to change as little as possible here to fix the bug.